readthedocs/builds/models.py

```diff
@@ -994,6 +1002,15 @@ def apply_action(self, version, match_result):
             raise NotImplementedError
         action(version, match_result, self.action_arg)
 
+    def get_description(self):
+        if self.description:
+            return self.description
+        return f'{self.get_action_display()}'
+
+    @property
+    def edit_url(self):
+        raise NotImplementedError
```

Comment:
Why do we have this?

Reply:
oops, that was part of another PR, not here